src/components/Question.js

```diff
@@ -119,19 +126,20 @@ class Question extends Component {
               Submit
             </button>
           </form>
-          {this.state.error && (
             <div
               style={{
                 position: "absolute",
                 left: "0",
                 right: "0",
                 bottom: "2px",
-                fontSize: "14px"
+                fontSize: "18px",
+                fontWeight: "700"
               }}
             >
-              *Please Select an Answer*
+              {this.state.error && (<div>Please Select an Answer</div>)}
```

Comment:
In order to make sure we only render one message, I think this message should be determined before the return statement in the render method. Something like this:

```javascript
let message;
if (this.state.error) {message = "Please select an answer"}
else if (this.state.corrrectAnswer) {message = "Correct!"}
else {message = "Incorrect"}

return (
  . . .
  <p>{message}</p>
  . . .
)
```